Fluent/Controls/RibbonTabItem.cs

```diff
@@ -609,7 +609,11 @@ protected override void OnMouseLeftButtonDown(MouseButtonEventArgs e)
 
                 if (this.TabControlParent != null)
                 {
-                    this.TabControlParent.IsMinimized = !this.TabControlParent.IsMinimized;
+                    var canMinimize = this.FindParentRibbon().CanMinimize;

```

Comment:
The TabControl mirrors CanMinimize so we should use that one here instead of looking at the ribbon.